README.md

```diff
@@ -0,0 +1,38 @@
+# deno_postgres_interop
+[![style: solid](https://img.shields.io/badge/style-solid-orange)](https://pub.dev/packages/solid_lints)
+[![$solid_lints](https://nokycucwgzweensacwfy.supabase.co/functions/v1/get_project_badge?projectName=deno_postgres_interop)](https://www.worklog.ai)
+
+This package was created to be used in supabase edge functions
+because existing db-driver/orm packages depend on dart:io.
+It should also work for other cases with dart2js.
+
+## Usage
+1. Compile the code that uses this package with:
+   ```bash
+   dart run edge build supabase_functions
+   ```
```

Comment:
Maybe some steps to create function or we can give some more basic guidance on what are the steps to add it to your project & where to look for API.

Reply:
> we can give some more basic guidance on what are the steps to add it to your project

Added that

> Maybe some steps to create function

The function is created as any other supabase edge function, so I think we shouldn't add it in our readme

Reply:
> where to look for API.

not sure if I understand this part. Is it about the JS part?
I added docs for everything and they point to the js lib docs
That can also be used to add necessary imports to compiled js